### ISSUE TITLE
Very quick but VERY URGENT Question - Please close upon answering

### ISSUE THREAD

**Issue by pGit1** [bethgelab/foolbox#160]
```
import foolbox
import keras
import numpy as np
from keras.applications.resnet50 import ResNet50

# instantiate model
keras.backend.set_learning_phase(0)
kmodel = ResNet50(weights='imagenet')
preprocessing = (np.array([104, 116, 123]), 1)
fmodel = foolbox.models.KerasModel(kmodel, bounds=(0, 255), preprocessing=preprocessing)

# get source image and label
image, label = foolbox.utils.imagenet_example()

# apply attack on source image
attack = foolbox.attacks.FGSM(fmodel)

fgsm_adversarials = []
for image, label in zip(some_training_images, corresponding_labels):
    adversarial = attack(image[:,:,::-1], label)
    fgsm_adversarials.append(adversarial)

```

does the output in the list fgsm_adversarials need to have the axes reversed before being fed to my pre proc function for resnet50? I usually flip to BGR then normalize regular images. Do I need to do this the fgsm_adversarials images in the list as well or is the output in RGB already?

**Comment by jonasrauber** [comment]
If your model expects BGR, then you should pass BGR images to the attack (which is what you seem to be doing by changing RGB images to BGR images using `image[:,:,::-1]`). Obviously, Foolbox has no knowledge whether the model expects RGB and you pass RGB, or whether the model expects BGR and you pass BGR. Accordingly, the adversarial that is returned by the attack will have the same format as the image that is passed to the attack, so in your case, `image[:,:,::-1]` and `adversarial` have the same color channel ordering (e.g. BGR in your case). So to get RGB adversarials, you need to reverse the channel order again.

**Comment by pGit1** [comment]
Perfect Thank you sir!!
